includes/class-wc-stripe-payment-tokens.php

```diff
@@ -98,37 +98,21 @@ public function woocommerce_get_customer_payment_tokens( $tokens, $customer_id,
 				$stored_tokens[] = $token->get_token();
 			}
 
-			if ( 'stripe' === $gateway_id ) {
-				$stripe_customer = new WC_Stripe_Customer( $customer_id );
-				$stripe_sources  = $stripe_customer->get_sources();
+			$stripe_customer = new WC_Stripe_Customer( $customer_id );
 
-				foreach ( $stripe_sources as $source ) {
-					if ( isset( $source->type ) && 'card' === $source->type ) {
-						if ( ! in_array( $source->id, $stored_tokens ) ) {
-							$token = new WC_Payment_Token_CC();
-							$token->set_token( $source->id );
-							$token->set_gateway_id( 'stripe' );
-
-							if ( 'source' === $source->object && 'card' === $source->type ) {
-								$token->set_card_type( strtolower( $source->card->brand ) );
-								$token->set_last4( $source->card->last4 );
-								$token->set_expiry_month( $source->card->exp_month );
-								$token->set_expiry_year( $source->card->exp_year );
-							}
+			if ( 'stripe' === $gateway_id ) {
+				$payment_methods  = $stripe_customer->get_payment_methods( 'card' );
```

Comment:
An extra space here gives me a phpcs warning and also in `in_array` because it's not strict. Feel free to ignore.